packages/gui/src/components/inputs/Dimension/Input.tsx

```diff
@@ -68,6 +70,7 @@ export const DimensionInput = ({
     themeValues.length > 0 && 'theme',
     ...units,
     keywords.length > 0 && 'keyword',
+    UnitlessUnits.Calc,
```

Comment:
I think this is ok here ?
My thinking is that anything that can use `calc` needs to supply either a `number` or a `<number><unit>` in the form of `<length>`, `<frequency>`, `<angle>`, `<time>`. So they'll likely use the dimension input in each case.